pyroengine/pi_utils/check_internet_connection.sh

```diff
@@ -36,7 +36,5 @@ then
     exit 0;
 fi;
 
-python3 /home/pi/pyro-engine/pyroengine/pi_utils/reboot_router;
-sleep 60;
```

Comment:
no longer relevant?

Reply:
no this was to reboot the wifi router that we are not using anymore